apps/web/pages/getting-started.tsx

```diff
@@ -377,12 +377,12 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
                   </label>
                   <p className="text-sm leading-tight text-gray-500 dark:text-white">
                     {t("current_time")}:&nbsp;
-                    <span className="text-black">{dayjs().tz(selectedTimeZone).format("LT")}</span>
+                    <span className="text-black">{dayjs().tz(dayjs.tz.guess()).format("LT")}</span>
```

Comment:
The whole point of having this here is for the user to confirm that their selected timezone matches up with the time on their computer, so this defeats the whole purpose.